modules/nf-core/sambamba/sort/main.nf

```diff
@@ -0,0 +1,38 @@
+process SAMBAMBA_SORT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::sambamba=1.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    'https://depot.galaxyproject.org/singularity//sambamba:1.0--h98b6b92_0':
+    'quay.io/biocontainers/sambamba:1.0--h98b6b92_0' }"
+
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("*.bam"), emit: bam
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    
```

Comment:
Can you add a check for file name collisions? See [here](https://nf-co.re/docs/contributing/modules#naming-conventions) part 6 for more info

Reply:
Its done! But here I would like to mention that, by default, sambamba_sort create a file with suffix "sorted.bam"

Reply:
Oh okay can you rename the output file to remove that?

Reply:
I can rename it , but may I ask the reason?

Reply:
Of course! We try to define the output file names exactly with `${prefix}.${extension}` This way you can know exactly how your file will be called :)

Reply:
This also is the most flexible to pipeline developers

Reply:
Okay, thanks for the quick response, but as "-o" option, which takes the output file name is optional, how should I go about changing the file name? Using "if else" condition? Any guidance to this will be highly appreciated?

Reply:
You can output them as normal and rename them using the `mv` command :)